### ISSUE TITLE
(Question) This project is abandoned?

### ISSUE THREAD

**Issue by hardBSDk** [libcala/cala#74]

**Comment by AldaronLau** [comment]
@hardBSDk no, it is not abandoned.  Updates to this crate are blocked on updates to dependencies, which have been in progress for a few years due to my limited time to work on them.
